ui/partials/main/jobs.html

```diff
@@ -25,33 +25,32 @@
     </span>
     <th-result-counts class="result-counts"></th-result-counts>
     <span class="result-set-buttons">
-      <span class="btn btn-sm btn-resultset cancel-all-jobs-btn"
+      <button class="btn btn-sm btn-resultset cancel-all-jobs-btn"
             tabindex="0" role="button"
             ng-attr-title="{{getCancelJobsTitle(resultset.revision)}}"
-            ng-show="currentRepo.repository_group.name == 'try' || user.is_staff"
```

Comment:
Just checking this is intentional, vs. previously hiding it. The change makes sense to me given our recent approach for that sort of thing.